event-based-simulators/main/scripts/ExportOrImportProfileData.py

```diff
@@ -0,0 +1,144 @@
+
+
+def listAlarms(c8y, childDevice, createFrom, createTo, filePath):
+    count = 0
+    for alarm in c8y.alarms.select(source=childDevice.id, created_after=createFrom, created_before=createTo):
+        print(
+            f"Found alarm id #{alarm.id}, severity: {alarm.severity}, time: {alarm.time}, creation time: {alarm.creation_time}, update time : {alarm.updated_time}\n")
+        print(f"{alarm.text}\n")
+        print(f"fragments: {list(alarm.keys())}\n")
+        count += 1
+        appendDataToJsonFile(alarm.to_json(), filePath, count)
+
+
+def listMeasurements(c8y, childDevice, createFrom, createTo, filePath):
+    # Create a count variable as a json/dict key to save json data
+    count = 0
+    for measurement in c8y.measurements.select(source=childDevice.id, after=createFrom, before=createTo):
+        print(f"Found measurement id #{measurement.id}\n, type: {measurement.type}\n")
+        count += 1
+        appendDataToJsonFile(measurement.to_json(), filePath, count)
+        # for measurementKey in measurement.fragments:
+        #    print(f"{measurementKey}: {measurement.fragments.get(measurementKey)}")
+
+
+def appendDataToJsonFile(jsonData, filePath, count, json_data={}):
+    try:
+        # Load content of existing json data file
+        with open(filePath, 'r') as f:
+            json_data = json.load(f)
+            print(json_data)
+    except:
+        print(f"Create new data json file {filePath}")
+
+    # Create new json file or add data to an existing json file
+    with open(filePath, 'w') as f:
+        json_data[f"{count}"] = jsonData
+        json.dump(json_data, f, indent=2)
+        print("New data is added to file")
+
+
+def createFilePathFromDateTime(DATA_TYPE):
+    # Check if folder containing data files exists and make one if not
+    if not os.path.exists('export_data'):
+        os.makedirs('export_data')
+    # dd/mm/YY H:M:S
+    dateTimeString = datetime.now().strftime(f"{DATA_TYPE}_%d_%m_%Y_%H_%M_%S")
+    relativeFilePath = f'export_data\{dateTimeString}.json'
+    filePath = os.path.join(os.path.dirname(__file__), relativeFilePath)
+    return filePath
+
+
+def chooseOneOfChoices(listOfChoices):
+    while True:
+        listToStringWithCommas = "', '".join(listOfChoices)
+        print(f"Type one of these words '{listToStringWithCommas}'")
+        inputString = input()
+        if inputString in listOfChoices:
+            break
+        else:
+            print(f"the choice {inputString} is not accepted, only input {listToStringWithCommas}")
+    return inputString
+
+
+def checkFileList():
+    if not os.path.exists('export_data'):
+        print("No folder with name export_data")
+    else:
+        onlyfiles = [f for f in os.listdir('export_data') if isfile(join('export_data', f))]
+        return onlyfiles
+
+
+# Main function to run the script
+if __name__ == '__main__':
+    c8y = ArgumentsAndCredentialsHandler.c8yPlatformConnection()
```

Comment:
- I like the idea to ask the user for input; however, for automation it is required to also start the script without any interaction
- so I guess it would be good to omit the question if the parameter is passed via command line or config file (Environment.py)